src/plugins/raven/actions/sendHeartbeat.js

```diff
@@ -37,7 +49,7 @@ export default (topic, etag = null) => async (dispatch, getState) => {
       ) {
         const nodeRef = NodeRef.fromString(topic);
         await swal.fire({
-          html: `This ${nodeRef.getLabel()} has been changed by another person or process.<br/>If you save, you may overwrite their changes.`,
+          html: `This ${nodeRef.getLabel()} has been changed by ${username} or process.<br/>If you save, you may overwrite their changes.`,
```

Comment:
@aloncohenn this could read `This article has been changed by Alon Cohen or process` which would be weird.

Reply:
i was back and forth on this, `This article has been changed by Alon Cohen or a system process` is what i had initially. if you like that better i can swap

Reply:
@aloncohenn sounds better if it's either `This article has been changed by Alon Cohen.` or `This article has been changed by an Unknown user or process`.